src/app/pages/system/general-settings/support/support-licensed/support-form-licensed.component.ts

```diff
@@ -273,9 +290,9 @@ export class SupportFormLicensedComponent implements FormConfiguration {
     this.screenshot['hasErrors'] = false;
     if (fileBrowser.files && fileBrowser.files[0]) {
       for (const browserFile of fileBrowser.files) {
-        if (browserFile.size >= 52428800) {
+        if (browserFile.size >= (this.maxUploadSizeMibs * 1024 * 1024)) {
           this.screenshot['hasErrors'] = true;
-          this.screenshot['errors'] = 'File size is limited to 50 MiB.';
+          this.screenshot['errors'] = this.translate.instant('File size is limited to ' + this.maxUploadSizeMibs + ' MiB.');
```

Comment:
`this.translate.instant('File size is limited to {maxUploadSizeMibs} MiB.', { maxUploadSizeMibs: 'this.maxUploadSizeMibs });`